draft-barnes-acme.md

```diff
@@ -852,10 +872,19 @@ Given a Challenge/Response pair, the server verifies the client's control of the
   * the path field is the path provided in the response.
 2. Verify that the resulting URI is well-formed.
 3. Dereference the URI using an HTTP or HTTPS GET request.  If using HTTPS, the ACME server MUST ignore the certificate provided by the HTTPS server.
-4. Verify that the Content-Type header of the response is either absent, or has the value "text/plain"
-5. Compare the entity body of the response with the nonce.  This comparison MUST be performed in terms of Unicode code points, taking into account the encodings of the stored nonce and the body of the request.
-
-If the GET request succeeds and the entity body is equal to the token, then the validation is successful.  If the request fails, or the body does not exactly match the token, then it has failed.
+4. Verify that the Content-Type header of the response is either absent, or has one of the values "application/jose" or "application/jose+json"
```

Comment:
I think that we should mandate the use of UTF-8, but that's a separate issue.  The server might reasonably support UTF-16 or UTF-32 in both endian forms for each, but...

![Support all the encodings!!!](http://memedad.com/meme/681576)